CHANGELOG.md

```diff
@@ -1,4 +1,33 @@
 ## 4.32.0 (Unreleased)
+
+NOTES:
+* Updated to Golang 1.18 ([#12246](https://github.com/hashicorp/terraform-provider-google/pull/12246))
+
+FEATURES:
+* **New Resource:** `google_dataplex_asset` ([#12210](https://github.com/hashicorp/terraform-provider-google/pull/12210))
+* **New Resource:** `google_gke_hub_membership_iam_binding` ([#12280](https://github.com/hashicorp/terraform-provider-google/pull/12280))
+* **New Resource:** `google_gke_hub_membership_iam_member` ([#12280](https://github.com/hashicorp/terraform-provider-google/pull/12280))
+* **New Resource:** `google_gke_hub_membership_iam_policy` ([#12280](https://github.com/hashicorp/terraform-provider-google/pull/12280))
+
+IMPROVEMENTS:
+* certificatemanager: added `state`, `authorization_attempt_info` and `provisioning_issue` output fields to `google_certificate_manager_certificate` ([#12224](https://github.com/hashicorp/terraform-provider-google/pull/12224))
+* compute: added `certificate_map` to `compute_target_https_proxy` resource ([#12227](https://github.com/hashicorp/terraform-provider-google/pull/12227))
+* compute: added validation for name field on `google_compute_network` ([#12271](https://github.com/hashicorp/terraform-provider-google/pull/12271))
+* compute: made `port` optional in `google_compute_network_endpoint` to be associated with `GCE_VM_IP` network endpoint groups ([#12267](https://github.com/hashicorp/terraform-provider-google/pull/12267))
+* container: added support for additional values `APISERVER`, `CONTROLLER_MANAGER`, and `SCHEDULER` in `google_container_cluster.monitoring_config` ([#12247](https://github.com/hashicorp/terraform-provider-google/pull/12247))
+* gkehub: added `policyControllerMonitoring` and `mutationEnabled` fields to resource `feature_membership` ([#12265](https://github.com/hashicorp/terraform-provider-google/pull/12265))
```

Comment:
```suggestion
* gkehub: added `monitoring` and `mutation_enabled` fields to resource `feature_membership` ([#12265](https://github.com/hashicorp/terraform-provider-google/pull/12265))
```
Looks like this PR only added the documentation of these two fields, the actual code change was added in a previous release.